auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/SessionHolder.java

```diff
@@ -26,15 +26,17 @@
  * Created by AR288503 on 9/6/2015.
  */
 public class SessionHolder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1118)